packages/composables/src/useListing.ts

```diff
@@ -4,6 +4,7 @@ import { getListingFilters } from "@shopware-pwa/helpers-next";
 import { useShopwareContext, useCategory } from "#imports";
 import ContextError from "./helpers/ContextError";
 import type { Schemas, RequestParameters } from "#shopware";
+import { useListingLoading } from "./useListingLoading";
```

Comment:
direct imports prevents overriding, that should be imported by `#imports`. But see the comment below, as `useListingLoading` probably should not exist